store/tikv/prewrite.go

```diff
@@ -238,3 +275,25 @@ func (c *twoPhaseCommitter) prewriteMutations(bo *Backoffer, mutations Committer
 	// `doActionOnMutations` will unset `useOnePC` if the mutations is splitted into multiple batches.
 	return c.doActionOnMutations(bo, actionPrewrite{}, mutations)
 }
+
+// Reprewrite the primary lock to fallback from async commit or 1PC
+func (c *twoPhaseCommitter) reprewritePrimary(bo *Backoffer) {
+	c.reprewrite.Add(1)
```

Comment:
Seems every time we enter the `reprewritePrimary` we will increase the wait value?

Reply:
Ah yes. It's a mistake. But it's strange that the corresponding unit test didn't fail.

Reply:
I see. I miss the test which fails several secondary prewrite.